pkg/metrics/telemetry.go

```diff
@@ -42,7 +51,7 @@ func InitKubecostTelemetry(config *MetricsConfig) {
 			Help: "kubecost_http_response_size_bytes Response size in bytes",
 		}, []string{"handler", "method", "code"})
 
-		prometheus.MustRegister(requestsCount, responseTime, responseSize)
+		prometheus.MustRegister(requestsCount, responseTime, responseSize, buildInfo)
```

Comment:
Did you mean to do this on the requestsCounts metric? I don't think that's necessary.

Reply:
@AjayTripathy I didn't think this impacted the other metrics. I was piggy backing off of the existing `prometheus.MustRegister` calls . If needed, I could move out buildInfo to it's own separate `prometheus.MustRegister` call

Reply:
Oh NVM I think it's fine.